src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/SecurityConfiguration.kt

```diff
@@ -96,7 +97,7 @@ class SecurityConfiguration<S : Session> : WebSecurityConfigurerAdapter() {
 
     return OAuth2UserService { userRequest ->
       val user = delegate.loadUser(userRequest)
-      val jwt = JwtDecoders.fromIssuerLocation(issuer).decode(userRequest.accessToken.tokenValue)
+      val jwt = (JwtDecoders.fromIssuerLocation(issuer) as JwtDecoder).decode(userRequest.accessToken.tokenValue)
```

Comment:
After bumping to v1.5 of Kotlin needed this as could not infer the returned type anymore.